libraries/AP_Scripting/docs/docs.lua

```diff
@@ -31,19 +33,21 @@ local enum_integer
 ---@operator shr(uint32_t_ud|integer|number): uint32_t_ud
 local uint32_t_ud = {}
 
--- create uint32_t_ud with optional value
----@param value? uint32_t_ud|integer|number
+-- Create a new uint32_t userdata object with an optional initial value.
+---@param value? uint32_t_ud|integer|number -- Optional initial value.
 ---@return uint32_t_ud
 function uint32_t(value) end
 
--- Convert to number
+-- Convert the uint32_t value to a standard Lua number (float).
 ---@return number
 function uint32_t_ud:tofloat() end
 
--- Convert to integer
+-- Convert the uint32_t value to a standard Lua integer.
 ---@return integer
 function uint32_t_ud:toint() end
 
+--- Custom 64-bit unsigned integer userdata type.
+-- Essential for high-resolution timestamps (like from `micros()`) or large IDs where standard Lua numbers would lose precision.
```

Comment:
`micros` returns uint32. This is used in utc epoch from the gps, and a few places in DroneCAN.